cjwkernel/thrift.thrift

```diff
@@ -192,6 +192,22 @@ union I18nArgument {
   3: double double_value
 }
 
+/** Source of a translatable string. 
+    If none of the values are set, this means it's coming from workbench itself
```

Comment:
Please follow to the docstring convention from the rest of this file. (By my count, this comment needs two more newlines and three more asterisks.)